vms/platformvm/camino_helpers_test.go

```diff
@@ -82,16 +83,13 @@ func defaultCaminoService(t *testing.T, camino api.Camino, utxos []api.UTXO) *Ca
 func newCaminoVM(t *testing.T, genesisConfig api.Camino, genesisUTXOs []api.UTXO, startTime *time.Time) *VM {
 	require := require.New(t)
 
-	vm := &VM{Config: defaultCaminoConfig()}
+	vm := &VM{Config: defaultCaminoConfig(t)}
 
 	db := memdb.New()
 	chainDB := prefixdb.New([]byte{0}, db)
 	atomicDB := prefixdb.New([]byte{1}, db)
 
-	if startTime == nil {
-		startTime = &defaultStartTime
-	}
```

Comment:
how come was this removed?

Reply:
Avax made changes in helpers, so vms now are created with specified phase. Highlighted lines are actually not the only ones, that we should change, I missed other places
Before this and missed changes, some of our tests were running with pre-banff phase, and since we weren't gating any of our logic behind banff (cause go-live is after-banff), everything was working fine. I fixed that now, see changes, now everything is running on latest fork (durango), since we're not testing anything earlier-phase specific